hypervisor/arch/x86/trusty.c

```diff
@@ -132,6 +132,10 @@ static void create_secure_world_ept(struct vm *vm, uint64_t gpa_orig,
 
 	/* Get the gpa address in SOS */
 	gpa = hpa2gpa(vm0, hpa);
+	if (gpa == INVALID_GPA) {
```

Comment:
this is vm0, so never got INVALID_GPA, right?

Reply:
I theory, this may happened. If we walk the reversed paging table, we could filter all the invalid hpa (such as the address over the real highest hpa or the address is unmapped from SOS - like the memory region used by trusty).